book/website/foreword/governance.md

```diff
@@ -114,24 +128,32 @@ Decisions at the maintainer level may include the following:
+
+Until November 2024, constitutional-level decision-making rested with the project leads, Kirstie Whitaker and Malvika Sharan.
+
+Since December 2024, the project delivery team, led by Malvika, has been supporting the formalisation of constitutional-level governance.
+Find details for _The Turing Way_ constitution-level roadmap in [issue #4014](https://github.com/the-turing-way/the-turing-way/issues/4014).
+
+A Steering Committee at this level is composed of chairs, representatives, or leads of Working Groups and projects funded or supported by _The Turing Way_.
 
-Strategic decisions and significant changes in the project, such as leadership, recruitment, goals, governance and funding.
-As of 2024, the constitutional level decision-making sits with the project leads, Kirstie Whitaker and Malvika Sharan.
-Their aspiration is to move decision-making power to the community through process development through which more community representation, perspectives and inputs can be brought to this level.
-Ideally, the chairs/representatives/leads of  WGs will join this level in the near future.
+Steering Committee members provide advice and ensure accountability for community- and maintenance-level efforts.
+They are responsible for escalating project- and community-wide decisions to the constitutional level and participating in the decision-making processes.
 
-In the future, we hope that we can work with the community to further design roles such as for community representatives to advise and build stronger accountability for decisions made at this level.
+They actively engage community members by transparently sharing discussions and outcomes from the constitutional level.
+They solicit feedback on decisions (such as, via GitHub issues and newsletters) and approve changes that improve project and community functioning.
 
 Decisions at constitutional level may include the following:
 - Providing leadership and strategic directions in the project.
-- Applying for core funding, and approving expenses.
-- Building clarity around budget and resources available for the maintainers and community members.
-- Proposing new directions for the community.
-- Process whereby the co-chairs of the WGs can represent community and WG interests at this level of decision-making.
-- Designing community representation roles such as through an Advisory Group.
-- Support each  WG in developing process documentation for their decision-making process, and identifying resources they need to continue their work.
-- Meeting other institutional obligations and communicating them transparently to the community.
+- Applying for core funding, recruiting staff and allocating budget.
+- Building clarity around resources available for the maintainers and community members.
+- Proposing directions for the community.
+- Process whereby the chair/representatives of the WGs can represent community and WG interests at this level of decision-making.
```

Comment:
```suggestion
```

I think this one sticks out - unless I've misunderstood the wording here doesn't describe a decision to be made, so I would suggest to delete this. It's part of the Steering Committee role but that's already covered above

Reply:
I am not sure what you mean here.